src/Microsoft.VisualStudio.AppDesigner/PropPages/PropertyControlData.vb

```diff
@@ -1726,6 +1726,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                     '  (other than cancel/checkout exceptions).
                     Descriptor.SetValue(Component, Value)
 
+                    ' Workaround until CPS is fixed to fire the value changed event on Descriptor.SetValue
+                    _valueChangedWasFired = Value.Equals(Descriptor.GetValue(Component))
```

Comment:
I'd like to keep this workaround until we all move to a newer build (probably next preview?). I'll revert it then.